net-kit/autogen.kit.d/base.yml

```diff
@@ -2627,4 +2627,189 @@ gitlab_freedesktop_dirlisting:
               rmdir "${ED}"/var{/lib{/NetworkManager,},} || die
             }
 
+libcupsfilters:
+  generator: builtin-github
+  template:
+    engine: helm
+
+  defaults:
+    files_dir: "{{ .Values.pn }}"
+    template: ../../templates/simple.tmpl
+    category: net-print
+    github:
+      user: OpenPrinting
+      repo: libcupsfilters
+    assets:
+      - name: '{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+        url: 'https://github.com/OpenPrinting/{{ .Values.pn }}/releases/download/{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+
+  packages:
+    - libcupsfilters:
+        transform:
+          - kind: string
+            match: 'libcupsfilters '
+            replace: ''
+          - kind: string
```

Comment:
I think that you can exclude release rc*|beta|alpha directly with exclude and/or as regex in github section. It's better just use stable release.